### DISCUSSION TITLE
💥 v0.0.2 Add TPO for yolov11

### DISCUSSION THREAD

**Discussion by ANGJustinl** [ANGJustinl/gcxl_2025#14] [Ideas]
# **Add TPO for yolov11**

## New Contributors
* @qilinbro made their first contribution in https://github.com/ANGJustinl/gcxl_2025/pull/1
* @ANGJustinl made their first contribution in https://github.com/ANGJustinl/gcxl_2025/pull/4

**Full Changelog**: https://github.com/ANGJustinl/gcxl_2025/commits/v0.0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/ANGJustinl/gcxl_2025/releases/tag/v0.0.2'>💥 v0.0.2 Add TPO for yolov11</a>.</em>

**Comment by qilinbro** [comment]
good👍


---Original---
From: ***@***.***&gt;
Date: Fri, Jan 3, 2025 19:50 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [ANGJustinl/gcxl_2025] 💥 v0.0.2 Add TPO for yolov11 (Discussion #14)




 
Closed #14 as resolved.
 
&mdash;

**Reply by ANGJustinl** [reply]
你吓我一跳
